### PR TITLE
Add API docs to Delius and NOMIS containers

### PR DESCRIPTION
**Pull request by sldblog**
## What does this pull request do?

Adds API docs to Delius `community-api` (related to https://github.com/ministryofjustice/community-api/pull/242)
Adds API docs to NOMIS `prisoner-offender-search` and `prison-api`s.

## What is the intent behind these changes?

To make sure we have as many APIs documented and published in #61 as possible.

### CODE REVIEW

File: src/main/kotlin/model/Delius.kt [hunk 1]
```diff
@@ -44,6 +44,7 @@ class Delius private constructor() {
         "Community API",
         "API over the nDelius DB used by HMPPS Digital team applications and services", "Java"
       ).apply {
+        APIDocs("https://community-api.test.delius.probation.hmpps.dsd.io/swagger-ui.html").addTo(this)
```
Comment (philroffe) [inline src/main/kotlin/model/Delius.kt:47]: I notice this is a test URL, where-as the other swagger docs are production.  
Also it's not accessible unless on the MoJ VPN - maybe not an issue for a dev, but just an observation about the availability of the swagger docs.

Comment (philroffe) [reply src/main/kotlin/model/Delius.kt:47]: I'll approve anyway as it is factually correct and what we use right now, but maybe highlights a few inconsistencies we can look at in future.

**Review by philroffe — APPROVED** [review]
Approved.  Might be useful to try to drive some consistency in our swagger docs in future (different test envs etc).

File: src/main/kotlin/model/Delius.kt (hunk 1 above)
Comment (sldblog) [reply src/main/kotlin/model/Delius.kt:47]: 👍 Yes, unfortunately, this is the one that's widely shared with everyone at the moment (as far as I can see).

I want to go through the whole lot and settle on pre-production or production URLs at some point next week. I'll make an issue out of this with a deadline

Comment (sldblog) [reply src/main/kotlin/model/Delius.kt:47]: Opened #64.

**Comment by sldblog** [comment]
Thank you! 🙇‍♂️

**Pull request merged**